app/src/main/java/de/westnordost/streetcomplete/settings/SettingsFragment.java

```diff
@@ -53,10 +53,12 @@ public void onStart()
 
 	private void updateOsmAuthSummary()
 	{
-		Preference oauth = getPreferenceScreen().findPreference("oauth");
+		Preference oauth = getPreferenceScreen().findPreference(Prefs.OAUTH);
+		String username = prefs.getString(Prefs.OSM_USER_DISPLAY_NAME, null);
 		if (oAuth.isAuthorized())
 		{
-			oauth.setSummary(R.string.pref_title_authorized_summary);
+			String summary = String.format(getResources().getString(R.string.pref_title_authorized_username_summary), username);
+			oauth.setSummary(summary);
 		}
```

Comment:
Do you want me to revert `.findPreference(Prefs.OAUTH)` to `.findPreference("oauth")` ?